src/libs/actions/IOU.js

```diff
@@ -2973,6 +2977,18 @@ function getIOUReportID(iou, route) {
     return lodashGet(route, 'params.reportID') || lodashGet(iou, 'participants.0.reportID', '');
 }
 
+/**
+ *  Helper function to get the receipt error for money requests, or the generic error if there's no receipt
+ *
+ * @param {Object} transaction
+ * @returns {Object}
+ */
+function getReceiptError(transaction) {
```

Comment:
The idea is to pass the receipt to it, and to also use the same method for the 2 last blocks.

Reply:
Yep. I'm having issues testing today (onyx is doing weird stuff). I'll try again tomorrow!